<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Failure</title>
    <!-- Bootstrap -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
  </head>
  <body>
    <div class="p-5 mb-4 text-bg-dark">
      <div class="container-fluid py-5">
        <h1 class="display-5 fw-bold">Uh Oh!!!</h1>
        <p class="col-md-8 fs-4">There was an error with signing up, please try again!</p> 
        <br>  
        <form action="/failure" method="POST">
          <button class="btn btn-lg btn-outline-light border-white" type="submit">Try Again!</button>
        </form> 
      </div>
    </div>
  </body>
</html>

<!-- When the button is clicked, it will trigger a POST request to the failure route, which will be caught by our server on line 67 -->
<!-- This will redirect to the home route for user to try again -->